browser/resources/extensions/extensions_resources.grd

```diff
@@ -10,12 +10,13 @@
   </outputs>
   <release seq="1">
     <structures>
-      <structure name="IDR_EXTENSIONS_BRAVE_ITEM_LIST_MORE_ITEMS_JS" file="brave_item_list_more_items.js" type="chrome_html" preprocess="true" />
-      <structure name="IDR_EXTENSIONS_BRAVE_ITEM_LIST_MORE_ITEMS_HTML" file="brave_item_list_more_items.html" type="chrome_html" preprocess="true" allowexternalscript="true" />
       <!-- included in chromium_src/chrome/browser/ui/webui/print_preview/print_preview_ui.cc -->
       <structure name="IDR_BRAVE_PDF_VIEWPORT_JS" file="js_override/pdf/viewport.js" preprocess="true" type="chrome_html" />
     </structures>
     <includes>
+      <include name="IDR_EXTENSIONS_BRAVE_ITEM_LIST_MORE_ITEMS_JS"
```

Comment:
do we not need the html file anymore? Should it be deleted?

Reply:
We still need the HTML file - it serves as an input to the modulizer here
https://github.com/brave/brave-core/pull/4374/files#diff-e7521305518dc98a36d472d2547ececcR67 (browser/resources/extensions/BUILD.gn:67).